### ISSUE TITLE
Zoom in and out with ctrl+scroll

### ISSUE THREAD

**Issue by nagotown** [sdercolin/vlabeler#6]
Add the ability to zoom in and out with a keybind (commonly crtl+scroll)

**Comment by sdercolin** [comment]
Try shift+ctrl+scroll.
shift+scroll is horizontal scroll so with additional ctrl it becomes zoom in/out.
(Forgot to write it in README, added recently)

**Comment by nagotown** [comment]
crtl+scroll is usually what it is in DAWs to zoom as far as I know, so it was confusing

**Comment by sdercolin** [comment]
Tried on my DAW:
```
scroll -> vertical scroll
ctrl + scroll -> vertical zoom
shift + scroll -> horizontal scroll
ctrl + shift + scroll -> horizontal zoom
```

So on vLabeler,
we do exatcly the same things with 
```
shift + scroll -> horizontal scroll
ctrl + shift + scroll -> horizontal zoom
```
and since we don't need vertial scrolling,
we change it to switching entries
```
scroll -> prev/next entry
ctrl + scroll -> prev/next sample
```

**Comment by nagotown** [comment]
Is it planned to be able to set custom keybinds?

**Comment by sdercolin** [comment]
yes, but not with a high priority.

**Comment by sdercolin** [comment]
Custom keymaps are available in `1.0.0-alpha15`
